src/components/home-children/ReviewList.js

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+import { getAllReviews } from "../../utils/api";
+import ReviewCard from "./review-list-children/ReviewCard";
+
+const ReviewList = () => {
+  const [reviewList, setReviewList] = useState([]);
+
+  useEffect(() => {
+    getAllReviews().then((allReviews) => {
+      setReviewList(allReviews);
+    });
+  }, []);
+
+  return (
+    <>
+      <ul>
```

Comment:
Looks good, although semantically, you have a ul, but then divs inside it. 

You probably dont need a ul, it could just be a div. OR ul, with li's